core/src/table/table.tsx

```diff
@@ -33,6 +33,11 @@ export type TableSize = {
 	cell: string;
 };
 
+export enum TableFixedColumn {
+	First = 1 << 0,
+	Last = 1 << 1,
+}
+
```

Comment:
Thank you @ZeroX-DG ! This is smart and I learned a lot by reading this! However, truth is this is a little bit over our maintenance level :( in other words, we don't feel comfortable building up on this technique, and we think our future contributors would be so. We're trying to keep the complexity of this project pretty low, and this bit-wise technique is rather unfamiliar to many web developers.

I hope you understand this. This is a great PR, especially the CSS part you did exactly what we expected. However, we're requesting this prop to be changed to object-based type, such as:

```tsx
interface TableFixed {
  header?: boolean;
  first?: boolean;
  last?: boolean;
}
```

We know that this breaks the previous API but it's the compromise we'd accept